lib/services.js

```diff
@@ -90,11 +91,27 @@ module.exports = function generateServices(app, options) {
     { encoding: 'utf-8' }
   );
 
+  var commonModelPrefix = 'LoopBack';
+  if (options.namespaceCommonModels) {
+    commonModelPrefix = options.ngModuleName + options.namespaceDelimiter;
+    if (options.namespaceDelimiter === '.') {
+      return 'throw new Error("Unsupported delimiter for ' +
+        'namespacing common models.");';
```

Comment:
```diff
- return 'throw new Error("Unsupported delimiter "." for ' +
+ throw new Error('Unsupported delimiter "." for ' +
```

If we keep your original code, then I think you need to escape `"."` as `\".\"`. I am surprised this was not caught by the tests.